utils/channel_utils.jsx

```diff
@@ -2,17 +2,15 @@
 // See LICENSE.txt for license information.
 
 import {haveIChannelPermission} from 'mattermost-redux/selectors/entities/roles';
-import {getRedirectChannelNameForTeam as getRedirectChannelNameForTeamRedux} from 'mattermost-redux/selectors/entities/channels';
 import Permissions from 'mattermost-redux/constants/permissions';
 
-import store from 'stores/redux_store.jsx';
 import Constants from 'utils/constants';
 import * as Utils from 'utils/utils.jsx';
 
-export function canManageMembers(channel) {
+export function canManageMembers(state, channel) {
```

Comment:
We might want to consider moving may of these utilities into `selectors`, but that doesn't need to be done now since there's already enough changing here.